lib/ignore/ignore.go

```diff
@@ -517,7 +533,33 @@ func parseIgnoreFile(fs fs.Filesystem, fd io.Reader, currentFile string, cd Chan
 			continue
 		}
 
-		line = filepath.ToSlash(line)
+		trimmed := strings.ReplaceAll(strings.ReplaceAll(line, " ", ""), "\t", "")
+		if strings.HasPrefix(trimmed, escapePrefix) {
+			// Silently ignore multiple #escape='s in a file.
```

Comment:
I think this should be an error rather than a silent skip, personally

Reply:
Fixed in the latest push.